src/Microsoft.AspNet.Mvc.ViewFeatures/AutoValidateAntiforgeryTokenAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNet.Mvc.ViewFeatures
+{
+    /// <summary>
+    /// An attribute that causes validation of antiforgery tokens for all unsafe HTTP methods.
```

Comment:
http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html "Safe" is defined here. I'm using "unsafe" to mean the negation.

> Can it instead just list the exact methods it checks?

HTTP methods are arbitrary text yo. We can specify the ones we **don't** check.